tests/reg_tests/test_DVConstraints.py

```diff
@@ -97,29 +104,31 @@ def generic_test_base(DVGeo, DVCon, handler, checkDerivs=True, fdstep=1e-4):
     return funcs, funcsSens
 
 
-@parameterized_class(
-    [
-        {
-            # Standard one-level FFD
-            "name": "standard",
-            "child": False,
-            "multi": False,
-        },
-        {
-            # Deforming child FFD with a stationary parent FFD
-            "name": "child",
-            "child": True,
-            "multi": False,
-        },
-        {
-            # One deforming component FFD and a stationary component FFD
-            # The components do not intersect
-            "name": "multi",
-            "child": False,
-            "multi": True,
-        },
-    ]
-)
+test_params = [
+    {
+        # Standard one-level FFD
+        "name": "standard",
+        "child": False,
+        "multi": False,
+    },
+    {
+        # Deforming child FFD with a stationary parent FFD
+        "name": "child",
+        "child": True,
+        "multi": False,
+    },
+    {
+        # One deforming component FFD and a stationary component FFD
+        # The components do not intersect
+        "name": "multi",
+        "child": False,
+        "multi": True,
+    },
+]
+
+# Skip multi component test if DVGeometryMulti cannot be imported (i.e. pySurf is not installed)
+@unittest.skipUnless(pysurfInstalled, "Multi-component tests require pySurf")
```

Comment:
I believe this would skip all the tests if pysurf is not installed? The previous if statement included `self.multi` as well. Now it's working because it's a public repo so `pySurfInstalled` is always true.

I believe the logic should be `skipUnless(pysurfInstalled or not self.multi)`?